### ISSUE TITLE
Receiving the following error when alt+clicking more than one token in a scene...

### ISSUE THREAD

**Issue by Zapgun** [JacobMcAuley/foundry-patrol#6]
Default DnD5e world with no other modules.

Create a scene, add two tokens, alt+click one, then alt+click the other, receive the following error:

`foundry.js:2420 Uncaught TypeError: Cannot read property 'filter' of undefined
    at KeyboardManager._onAlt (foundry.js:2420)
    at KeyboardManager._handleKeys (foundry.js:2276)
    at KeyboardManager._onKeyUp (foundry.js:2242)
    at foundry.js:2112
_onAlt @ foundry.js:2420
_handleKeys @ foundry.js:2276
_onKeyUp @ foundry.js:2242
(anonymous) @ foundry.js:2112`

**Comment by Norc** [comment]
I'm still getting this issue in Foundry Token Patroller 2.1.3 with 0.60 (dnd 5E 0.92 if that matters). Confirmed it happens with only Foundry Token Patroller enabled.

This is more of a severe issue than it may seem on reading the issue log. The error KEEPS happening as long as Alt is held down, to the tune of 100+ errors in a few seconds. This has the practical effect of bringing my GM copy of Foundry to a rendering standstill while alt is pressed.

Basically, this could be titled "Holding Alt Key severely lags GM copy of Foundry while Foundry Patrol is Enabled."

Thanks for looking into this!!!

**Comment by JacobMcAuley** [comment]
Thanks for the information!
It should be resolved now, let me know if it continues to occur.
